code/game/machinery/computer/atmos_computers/_air_sensor.dm

```diff
@@ -72,8 +85,110 @@
 		output.chamber_id = chamber_id
 		GLOB.objects_by_id_tag[CHAMBER_OUTPUT_FROM_ID(chamber_id)] = output
 		balloon_alert(user, "connected to output")
-	else
-		multi_tool.buffer = src
-		balloon_alert(user, "added to multitool buffer")
 
 	return TRUE
+
+/obj/item/air_sensor
+	name = "Air Sensor"
+	desc = "It's an switched off air sensor."
```

Comment:
"It's a switched-off air sensor." I'm not sure about this for a description, actually. Maybe something like "A device designed to detect gases and their concentration in an area." We need to describe what an air sensor is to players who maybe don't know.

Reply:
Yeah i have no clue how to be more descriptive, i'll do that